spring/boot2-webflux-autoconfigure/src/test/java/com/linecorp/armeria/spring/web/reactive/ReactiveWebServerLoadBalancerInteropTest.java

```diff
@@ -46,46 +54,91 @@
     static class TestConfiguration {
         @RestController
         static class TestController {
-            @GetMapping("/api/ping")
+            @GetMapping("/controller/api/ping")
             Mono<String> ping() {
                 return Mono.just("PONG");
             }
         }
+
+        @Bean
+        RouterFunction<ServerResponse> routerFunction() {
+            return route(GET("/router/api/poke"), request -> ok().build())
+                    .and(route(HEAD("/router/api/poke"), request -> ok().build()))
+                    .and(route(HEAD("/router/api/ping"), request -> ok().body(fromValue("PONG"))));
+        }
     }
 
     @LocalServerPort
     int port;
 
     @Test
-    void get() throws Exception {
-        try (Socket s = new Socket(NetUtil.LOCALHOST4, port)) {
-            s.setSoTimeout(10000);
-            final InputStream in = s.getInputStream();
-            final OutputStream out = s.getOutputStream();
-            out.write("GET /api/ping HTTP/1.0\r\n\r\n".getBytes(StandardCharsets.US_ASCII));
-
-            // Should not be chunked.
-            assertThat(new String(ByteStreams.toByteArray(in))).isEqualTo(
-                    "HTTP/1.1 200 OK\r\n" +
-                    "content-type: text/plain;charset=UTF-8\r\n" +
-                    "content-length: 4\r\n\r\n" +
-                    "PONG");
-        }
+    void getToController() throws Exception {
+        // TODO: Need to assert that CancelledSubscriptionException is not propagated to HttpWebHandlerAdapter
```

Comment:
I wanted to assert that [HttpWebHandlerAdapter#handleUnresolvedError](https://github.com/spring-projects/spring-framework/blob/v5.2.7.RELEASE/spring-web/src/main/java/org/springframework/web/server/adapter/HttpWebHandlerAdapter.java#L271) is never invoked due to CancelledSubscriptionException,
but couldn't find an appropriate way to test it...
(I looked for a way to test logEvent of HttpWebHandlerAdapter's `log`, but failed to do that)

It would be very helpful if I can get some advices here